pkg/app/master/commands/cliprompt.go

```diff
@@ -210,6 +210,7 @@ var GlobalFlagSuggestions = []prompt.Suggest{
 	{Text: FullFlagName(FlagLogLevel), Description: FlagLogLevelUsage},
 	{Text: FullFlagName(FlagLog), Description: FlagLogUsage},
 	{Text: FullFlagName(FlagLogFormat), Description: FlagLogFormatUsage},
+	{Text: FullFlagName(FlagConsoleOutput), Description: FlagConsoleOutputUsage},
```

Comment:
also need the config for value completion (`CompleteContinueAfter` is a good example)

Reply:
To do after this

Reply:
done, do check, thanks a lot!